### PR TITLE
Refactor Shp2TempTableTask

### PR DESCRIPTION
**Pull request by antoniocarlon**
Refactor `Shp2TempTableTask` to `GeoFile2TempTableTask`
Closes https://github.com/CartoDB/bigmetadata/issues/549

### CODE REVIEW

File: tasks/au/geo.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 from lib.timespan import get_timespan
 
-from tasks.base_tasks import (ColumnsTask, DownloadUnzipTask, Shp2TempTableTask, SimplifiedTempTableTask, TableTask,
+from tasks.base_tasks import (ColumnsTask, DownloadUnzipTask, GeoFile2TempTableTask, SimplifiedTempTableTask, TableTask,
```
Comment (juanignaciosl) [inline tasks/au/geo.py]: There are still 4 mentions of `Shp2TempTableTask`, two minor ones at `example.rst` and `development.rst`, and two slightly more important at `test_tabletasks.py`. I'd change all of them, as they become confusing.

File: tasks/base_tasks.py [hunk 2]
```diff
@@ -577,47 +577,47 @@ def run(self):
...
             schema=schema)
-        for shp in shps:
+        for geofile in files:
             # First we  try to import using utf8 as encoding if not able we
             # fallback to the encoding passed as parameter
             cmd = self.build_ogr_command(encoding='utf-8',
                                          schema=schema,
                                          tablename=tablename,
-                                         shp=shp,
+                                         geofile=geofile,
```
Comment (juanignaciosl) [inline tasks/base_tasks.py]: Have you considered renaming the `geofile` option to `input` to match `ogr2ogr` syntax?

**Comment by antoniocarlon** [comment]
Done! @juanignaciosl another round?

**Comment by juanignaciosl** [comment]
`make -- docker-run au.geo.Geography --year 2011 --resolution MB` worked. I'd say that it's enough, so I'll hand this to you for production deploy. If you wanted more testing, let me know.

**Comment by antoniocarlon** [comment]
Perfect, thank you! I'm going to merge it to master and the rest of the pending PRs! :+1:

**Pull request merged**